pkg/storage/engine.go

```diff
@@ -1033,11 +1033,22 @@ type Engine interface {
 	// additionally returns ingestion stats.
 	IngestLocalFilesWithStats(
 		ctx context.Context, paths []string) (pebble.IngestOperationStats, error)
-	// IngestAndExciseFiles is a variant of IngestLocalFilesWithStats
-	// that excises an ExciseSpan, and ingests either local or shared sstables or
-	// both.
+	// IngestAndExciseFiles is a variant of IngestLocalFilesWithStats that excises
+	// an ExciseSpan, and ingests either local or shared sstables or both. It also
+	// takes the flag sstsContainExciseTombstone to signal that the exciseSpan
+	// contains RANGEDELs and RANGEKEYDELs.
+	//
+	// NB: It is the callers responsibility to ensure if
+	// sstsContainExciseTombstone is set to true, the snapshot must contain a
```

Comment:
terminology creep; in the storage package, `snapshot` has a different meaning. Say "ingestion sstables".

Reply:
nice catch, fixed! we do overload the term snapshot (pun intended) 😄